composer.json

```diff
@@ -4,12 +4,13 @@
     "type": "library",
     "license": "GPL-3.0",
     "suggest": {
-        "nunomaduro/larastan": "Provides phpstan extension specifically aimed towards Laravel"
+        "larastan/larastan": "Provides phpstan extension specifically aimed towards Laravel"
     },
     "conflict": {
-        "php": "<7.4",
+        "php": "<8.0",
         "friendsofphp/php-cs-fixer": "<3.11",
-        "nunomaduro/larastan": "<0.6",
+        "larastan/larastan": "<2.7.0",
+        "nunomaduro/larastan": "<2.7.0",
```

Comment:
hoort het geen conflict te zijn op alle versies, dus '*'? Want er is geen nieuwere versie dan 2.7.0 en volgens mij mag met <2.7.0 je wel gaan voor 'dev-master' of 'dev-main'.

Of helemaal geen conflict met de oude? Want je weet nooit of een 3rd party nunomaduro/larastan als dependency heeft.

Reply:
@pieterw2w Ja nice, ik heb er * van gemaakt. Als een andere package nunomaduro/larastan nog al dependency heeft kan deze versie nog niet geinstalleerd worden en blijf je op de oude versie.

Reply:
ja, dat snap ik, maar is dat wenselijk? Als een package nunomaduro/larastan include, voegt het iets toe wat de coding-standards niet gebruikt dus eigenlijk is het niet conflicting.
Het kan ook verwarrend zijn als je coding-standards als dependency hebt en je hem wilt updaten zal het niet snel duidelijk zijn waarom je deze laatste versie niet mag installeren (of nog erger: je denkt up to date te zijn, maar je bent het niet)

Ik vind het niet blocking voor nu, maar ik vraag me af of het conflict iets toevoegt.